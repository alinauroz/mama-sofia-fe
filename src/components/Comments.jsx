import React from 'react';
import { DiscussionEmbed } from 'disqus-react';

export default function Comments (props) {
    return (
        <div className = "comments" style={{width: "100%"}}>
            <DiscussionEmbed
                shortname="mamasofia"
                config={
                    {
                        url: window.location.href,
                        identifier: props.id,
                        title: props.name,
                        language: 'en-US' //e.g. for Traditional Chinese (Taiwan)	
                    }
                }
            />
        </div>
    )
}