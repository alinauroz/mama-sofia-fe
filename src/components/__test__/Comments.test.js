import React from 'react';
import ReactDOM from 'react-dom'
import Comments from './../Comments.jsx'
import renderer from 'react-test-renderer'

it ('comments renders without crashing', () => {
    const div = document.createElement("div");
    ReactDOM.render(<Comments />, div);
});

it ('matches snapshot', () => {
    const tree = renderer.create(
        <Comments
            name="Coconut & squash dhansak"
            id = "5f35abbdf05c76200731046a"
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
})