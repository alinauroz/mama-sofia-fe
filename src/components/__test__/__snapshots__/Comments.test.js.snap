// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="comments"
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    config={
      Object {
        "identifier": "5f35abbdf05c76200731046a",
        "language": "en-US",
        "title": "Coconut & squash dhansak",
        "url": "http://localhost/",
      }
    }
    id="disqus_thread"
    shortname="mamasofia"
  />
</div>
`;
